.gitignore

```diff
@@ -87,6 +87,9 @@ target/
 # Virtualenv
 venv
 
+# vs code
+.vscode
```

Comment:
No, only changes related to this PR should be here. Please make another issue for this

Reply:
okay